lib/classes/http.rb

```diff
@@ -11,9 +11,8 @@ def initialize(config)
       http_proxy = config[:http_proxy] || ENV['http_proxy']
       setup_proxy(http_proxy) if http_proxy
 
-      qb_params[:ticket] = auth_ticket config
-      qb_params[:apptoken] = config[:apptoken]
       qb_params[:dbid] = config[:dbid]
+      qb_params[:usertoken] = config[:usertoken]
     end
```

Comment:
@kalinchuk 
Usertoken can be used with any API that doesn’t post to a db/main URL, and also with API_GrantedDBs.
This method initialize is called for API that does not post to db/main URL ; for eg API like getSchema, API_DoQuery etc.
For those APIs query must be like below  without app token and ticket

<qdbapi>
<udata>mydata</udata>
<usertoken>user_token</usertoken>
</qdbapi>